src/main/java/com/PhotomosaicSara/image/Photomosaic.java

```diff
@@ -0,0 +1,34 @@
+package com.PhotomosaicSara.image;
+
+import javax.imageio.ImageIO;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Photomosaic {
+
+    private final PhotomosaicService photomosaicService;
+
+    public Photomosaic(PhotomosaicService photomosaicService) {
+        this.photomosaicService = photomosaicService;
+    }
+
+    public void process() throws IOException {
+        final var tileImages = photomosaicService.getImagesFromTiles(new File("source_images"));
+        final var inputImageFile = new File("mainImage.jpg");
+        final var inputImageParts = photomosaicService.getImagesFromInput(inputImageFile);
+        //  final Collection<BufferedImagePart> outputImageParts = Collections.synchronizedSet(new HashSet<>());
+        final ArrayList<BufferedImagePart> outputImageParts = new ArrayList<>();
```

Comment:
it can be final var:
final var outputImageParts = new ArrayList<BufferedImagePart>();